recipes/daymet/recipe.py

```diff
@@ -0,0 +1,51 @@
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+from pangeo_forge_cmr import get_cmr_granule_links
+
+from pangeo_forge_recipes import patterns
+import aiohttp
+import netrc
+
+# We need to provide EarthData credentials to fetch the files.
+# The credentials of the currently logged in user are used, and passed on to the cloud
+# as well when the operation is scaled out. This shall be automated with a machine identity
+# in the future.
+# go here to set up .netrc file: https://disc.gsfc.nasa.gov/data-access
+username, _, password = netrc.netrc().authenticators('urs.earthdata.nasa.gov')
+client_kwargs = {
+    'auth': aiohttp.BasicAuth(username, password),
+    'trust_env': True,
+}
+
+# Get the GPM IMERG Late Precipitation Daily data
+shortname = 'Daymet_Daily_V4_1840'
```

Comment:
Is the "1840" here specific to daily - North America? And other regions will have different numerical IDs here?

Reply:
Yes it is daily, but all spatial areas are distributed under that DOI.  The granule-level file name will distinguish the difference; na, pr, hi.  We just updates and 1840 is now 2129.  `https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=2129`